distributionviewer/core/static/css/distributionviewer.styl

```diff
@@ -45,21 +63,42 @@ a {
   margin-top: -($header-height)
   padding-top: $header-height;
 
+  @media all and (max-width: $min-sidebar-width) {
+    margin: $default-whitespace 0;
+    padding: 20px 20px 20px (20px - $mobile-filter-button-padding);
+  }
+
   ul {
+    @extends $clearfix;
+
     list-style: none;
     padding: 0;
     margin: 0;
   }
 
   li {
     margin-bottom: 10px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    @media all and (max-width: $min-sidebar-width) {
```

Comment:
I think I am going to rework these a bit and rely on flexbox some more (since this project is for internal consumption) but this is a great start. I found the media queries we did for marketplace had some good breakpoints which might make sense here.